content/contributing-code/usability/contents.lr

```diff
@@ -0,0 +1,60 @@
+- Indicate when (or if) you plan to commence screen recording.
+- Prepare tasks, one or more, for the user to carry out. It is helpful to write these tasks down on a shared document (we often use Google Slides), as they may include links, and can allow the user to reference the instructions during the task.
+
+**Prepare a consent form for screen recording.** Usability sessions are great, but they’re even better when they’re recorded. It allows you to focus more fully on the session during it, be less distracted by note taking, and not worry that you’re missing key insights. Ensure that your users sign this before commencing screen recording. (See the [form](https://drive.google.com/file/d/1XWt5QDu-IR76BwsPP7OidqNncha3rHbm/view) we use as an example.). If you opt to not do screen recording, we strongly recommend having a second person with you during the session whose sole role it is to take notes.
+
+**Schedule tests.** Promote with your social users, newsletter subscribers, announce it on your site, or get the word out some other way. We use Calendly for scheduling these, and we include a number of basic questions during the session sign-up process, to help us better prepare for tests. (See an example sign-up [here](https://calendly.com/cc-product-design/user-interview-30) ). Our typical requirements for participants are high speed internet access, a computer with a microphone, ability to screen share, and request for consent form to be signed. For anything new you want to get feedback on, it’s advisable to talk to at least three users.
+
+**Rehearse.** Before your first session, simulate a whole session, read the script, complete the task(s), and answer the final questions you plan to ask. Think about if there are additional things you want to ask. Be ready, knowing what you want to say and ask.
+
+**Carry out the test.** During the session, have your script open on one window, and a video call with screen share open on another window. We use Google Meet (Hangouts) for our video calls.
+
+Have a notes document prepared, or better yet, enlist someone else to join the call to take notes. If (and only if) your participant has consented to screen share being recorded, be ready with software that records the screen. We often use QuickTime (if on a Mac), it is simple and free. Another option is [OBS](https://obsproject.com/), which is free, open source, and cross platform.
+
+Remember that the point of the test is to ask questions and get feedback. It is not to educate, give guidance, or explain. Whether you are running a usability test, remember that you want  to see what the user will do to complete the task when no one is around to help them. Ask them to think out loud, so that you can better understand what is clear and what is confusing, and pay close attention to how the user interacts with the prototype. If something related to your research purpose catches your attention, dive into it by asking why.
```

Comment:
Typo (our fault) in the second sentence. Should say "Whenever you are..." and not "Whether you are...". There is also an extr aspace between "want" and "to" in that same sentence.